www/main/viewer/Slide/Slide.jsx

```diff
@@ -58,9 +67,12 @@ const Slide = ({ verseObj, nextLineObj, isMiscSlide, bgColor, updateVerseRef })
     }, 200);
 
     return () => clearTimeout(timeoutId);
-  }, [verseObj, isMiscSlide]);
+  }, [verseObj, isMiscSlide, akhandpatt]);
 
   useEffect(() => {
+    // Skip internal scrolling in akhand path mode since ShabadDeck handles it
+    if (akhandpatt) return;
```

Comment:
@amitojsingh The internal scrolling is needed imo, we are syncing the scroll, if we scroll in the viewer preview directly, it would update the scroll position in viewer window.